src/components/Cards.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Card } from './Card'
+import { ICardsProps } from '../models/propsModels.js'
+import './Cards.css'
+
+const Cards = ({ data }: ICardsProps) => {
```

Comment:
1. Conditional (Ternary) Operator: Sometimes using the ternary operator can make conditional rendering a bit cleaner.
2. Styling: Inline styles might be avoided for maintaining a cleaner JSX and separating concerns. It's often better to utilize CSS classes for styling.
3. Semantic HTML: Consider using more semantic HTML for better SEO and accessibility. E.g., using `<h1>` for error messages might not be semantically correct. Consider replacing it with a paragraph `<p>` or a specialized error component that encapsulates the styling and semantics.
```tsx
const Cards = ({ data }: ICardsProps) => {
    return data && data.length > 0 ? (
        <div className={styles.cards}>
            {data.map((promo) => (
                <Card key={promo.id} {...promo} />
            ))}
        </div>
    ) : (
        <p className={styles.notFound}>Not found</p>
    );
};